sdk/src/main/java/io/radar/sdk/RadarSettings.kt

```diff
@@ -192,14 +192,17 @@ internal object RadarSettings {
         getSharedPreferences(context).edit { remove(KEY_REMOTE_TRACKING_OPTIONS) }
     }
 
-    internal fun getForegroundService(context: Context): RadarTrackingOptions.RadarTrackingOptionsForegroundService? {
+    internal fun getForegroundService(context: Context): RadarTrackingOptions.RadarTrackingOptionsForegroundService {
```

Comment:
Since we migrated `trackingOptions.foregroundService` to `trackingOptions.foregroundServiceEnabled` and `Radar.setForegroundServiceOptions()`, we can just return the default `RadarForegroundServiceOptions` here, assuming we fix the logic here: https://github.com/radarlabs/radar-sdk-android/pull/251/files#diff-80113b4a9caa6ded13eca3973127e13b9053f26e6c32bb2f1b279809f5c446caR555. It also lets us move this fallback out of `RadarLocationManager`: https://github.com/radarlabs/radar-sdk-android/pull/251/files#diff-80113b4a9caa6ded13eca3973127e13b9053f26e6c32bb2f1b279809f5c446caL188